package.json

```diff
@@ -27,5 +32,8 @@
   "devDependencies": {
     "c8": "^7.12.0",
     "mocha": "^10.0.0"
+  },
+  "dependencies": {
+    "@types/node": "^18.8.2"
```

Comment:
`@types/*` are usually devDependencies.

Reply:
I'm not sure we should do that. Our lib can be used in production software and it will fail if it does not have its types.

Reply:
If someone gotta use our lib they only will need to have our `.d.ts` files and nothing more.

Reply:
> If someone gotta use our lib they only will need to have our `.d.ts` files and nothing more.

but our type definitions also use `@types/node`, so your `.d.ts` will say for example "*this function returns `Buffer`*" somewhere and without this dependency that type will not resolve. What do you think?

Reply:
Usually TS projects have `@types/node` but some not and i am saying to include to `devDependecies` only if someone want to contribute and would need the types of the external libraries.

Reply:
I created a fresh new test project and installed the branch directly from GitHub using npm.

With `@types/express` in production dependencies:
![Screenshot from 2022-10-26 20-41-43](https://user-images.githubusercontent.com/66572326/198098219-cceea1ab-451e-4464-b615-00ec52ae4010.png)
TSC is able to identify the correct type from express correctly.

With `@types/express` in dev deps:
![Screenshot from 2022-10-26 20-42-23](https://user-images.githubusercontent.com/66572326/198098657-0c35d81b-15d3-42ca-b0ec-4cfbecd4b3ad.png)
TS recognises it *as a type*, but it does not recognise its properties as an alias for the class (as seen in the first screenshot). Additionally, where types like `e.Request` were used, I was able to pass just any object and it would accept it as correct.

Anyways, I actually saw that `@types/node` is not needed on this branch so I am going to remove it altogether.